src/components/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+export default function Footer() {
+  return (
+    <footer className="bg-transparent position:relative flex flex-col justify-center items-center w-full before:content-[''] before:absolute before:w-full before:h-[14.7rem] before:bg-[url('/waves.svg')] before:bg-no-repeat before:bg-top before:-z-10">
```

Comment:
**:hammer: Ajuste**

`position:relative` não está aplicando nada na tag, pois não existe o prefixo de position no tailwind. Para usar relative, basta colocar `relative` direto, sem prefixo (ou remover caso não for usar)